python/HeaderService/hutils.py

```diff
@@ -384,17 +275,23 @@ def write_headers(self, filenames, MP=False, NP=2, md5=False):
                 write_header_string(arg)
         return
 
-    def write_header_emptyHDU(self, filename):
-
+    def write_header_fits(self, filename):
+        """Write a header file using the FITS format with empty HDUs"""
         data = None
         with fitsio.FITS(filename, 'rw', clobber=True, ignore_empty=True) as fits:
             for extname in self.HDRLIST:
                 hdr = copy.deepcopy(self.header[extname])
                 fits.write(data, header=hdr, extname=extname)
 
+    def write_header_string(self, filename):
+        """ Write header as string"""
+        self.string_header()
```

Comment:
It would look less like magic if the return value of this method was then printed out. `string_header()` would feel less magical I think if it was called `calculate_string_header()` so that it would more explicitly be updating `self`.  In this case it is much clearer to use the return value and not rely on the magic.

Reply:
Changed to function name to `calculate_string_header()` and use the return value rather than the `self` for clarity.